cmsplugin_blocks/models/card.py

```diff
@@ -79,11 +79,11 @@ class Card(SmartFormatMixin, CMSPlugin):
 
     content = models.TextField(
         _("Content"),
-        blank=False,
-        default="",
+        blank=True,
+        null=True,
```

Comment:
Nope, except in very specific context you don't allow for null value on a text column, only empty string.